src/Listener/RelatedModelsListener.php

```diff
@@ -87,7 +87,7 @@ public function publishRelatedModels($action = null, $entity = null)
         foreach ($models as $name => $association) {
             list(, $associationName) = pluginSplit($association->name());
             $viewVar = Inflector::variable($associationName);
-            if (array_key_exists($viewVar, $controller->viewVars)) {
+            if (array_key_exists($viewVar, (array)$controller->viewVars)) {
```

Comment:
All I am saying since someone did the work, and it clearly has no argument against it, it is not worth the negative discussion.
Merge, move on.